src/session.rs

```diff
@@ -0,0 +1,83 @@
+extern crate dirs;
```

Comment:
same here (not necessary in [non-`2015` edition](https://github.com/ducaale/ht/commit/f4332e23b5a425049c0f76d1e31136006262b683#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R5) Rust)